pandas/_version.py

```diff
@@ -311,10 +307,10 @@ def render_pep440_pre(pieces):
     if pieces["closest-tag"]:
         rendered = pieces["closest-tag"]
         if pieces["distance"]:
-            rendered += ".post.dev%d" % pieces["distance"]
+            rendered += f".post.dev{pieces['distance']}"
```

Comment:
I think you need the decimal integer format specifier here to keep this equivalent, unless distance is guaranteed to be an int

Reply:
I think you might be right, fixed all occurrences.